.github/workflows/ci-conan.yml

```diff
@@ -19,21 +19,22 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-18.04]
+        os: [ubuntu-latest]
         build_type: [Debug, Release]
-        compiler_version: [7, 8, 9]
+        compiler_version: [9]
         compiler_libcxx: [libstdc++11]
         option_shared: ['True', 'False']
```

Comment:
I suggest using containers when building. This configuration (gcc 9 and ubuntu 22) is unfortunate, since distributions using gcc 9 have an older glibc version than ubuntu 22 and pesky glibc issues are likely.
conanio have (almost) ready-made containers for various compiler versions.

```
container:
      image: conanio/gcc${{ matrix.compiler_version }}-ubuntu18.04
      options: -u 0
```
You would need to upgrade conan and either use `conan config install` or `conan profile detect` to setup.

Reply:
Also note that many conan environment variables, e.g. ` CONAN_NON_INTERACTIVE` are ignored with conan 2 and should be configured with `core:non_interactive = False` in `global.conf` . I think in this case it does not make a difference.

Reply:
I agree, and we actually do use containers in our main repo, libcosim. But since this is going to be just a stop-gap solution until you get [fmilibrary into conan-center](https://github.com/conan-io/conan-center-index/pull/20256), let's just leave it as-is for now.